server/api/categories.go

```diff
@@ -521,3 +523,125 @@ func (a *API) handleReorderCategoryBoards(w http.ResponseWriter, r *http.Request
 	jsonBytesResponse(w, http.StatusOK, data)
 	auditRec.Success()
 }
+
+func (a *API) handleHideBoard(w http.ResponseWriter, r *http.Request) {
```

Comment:
This needs to be added to the [client](https://github.com/mattermost/focalboard/blob/main/server/client/client.go) and a integration test added.

Reply:
Done.